runtime/rasdump/dmpmap.c

```diff
@@ -121,7 +127,33 @@ static const DgDumpEnvSwitch dgSwitches[] = {
 			}
 };
 
-static const int numDumpEnvSwitches = ( sizeof(dgSwitches) / sizeof(DgDumpEnvSwitch) );
+typedef struct OomDefaultSetting
+{
+	const char *typeString;
+	const char *onOpt;
+} OomDefaultSetting;
+
+static const OomDefaultSetting oomDefaultTable[] = {
+		{
+				HEAP_KIND,
+				SYSTHROW_OOM_1_4
+		},
```

Comment:
Please reduce the indentation (one tab per level is sufficient).

Reply:
Will do.  FYI, that was the Eclipse-supplied indentation.